tests/Jellyfin.Server.Implementations.Tests/Library/PathExtensionsTests.cs

```diff
@@ -11,6 +11,17 @@ public class PathExtensionsTests
         [InlineData("Superman: Red Son - tt10985510", "imdbid", "tt10985510")]
         [InlineData("Superman: Red Son", "imdbid", null)]
         [InlineData("Superman: Red Son", "something", null)]
+        [InlineData("Superman: Red Son [imdbid1=tt11111111][imdbid=tt10985510]", "imdbid", "tt10985510")]
+        [InlineData("Superman: Red Son [tmdbid=618355][imdbid=tt10985510]", "imdbid", "tt10985510")]
+        [InlineData("Superman: Red Son [tmdbid=618355][imdbid=tt10985510]", "tmdbid", "618355")]
+        [InlineData("[tmdbid=618355]", "tmdbid", "618355")]
+        [InlineData("tmdbid=618355][tmdbid=618355]", "tmdbid", "618355")]
+        [InlineData("[tmdbid=618355]tmdbid=618355]", "tmdbid", "618355")]
```

Comment:
might want the test to make sure we're picking up the _first)_value so make the second one different
`[InlineData("tmdbid=618777][tmdbid=618355]", "tmdbid", "618777")]`